lib/mbta_metro/components/list.ex

```diff
@@ -0,0 +1,30 @@
+defmodule MbtaMetro.Components.List do
+  @moduledoc false
+
+  use Phoenix.Component
+
+  @doc """
+  Renders a list of items.
+
+  ## Examples
+
+      <.list>
+        <:item>Foo</:item>
+        <:item>Bar</:item>
+        <:item>Baz</:item>
+      </.list>
+  """
+  attr :class, :string, default: ""
+
+  slot :item, required: true
+
+  def list(assigns) do
+    ~H"""
+    <ul class={"max-w-xs flex flex-col divide-y divide-slate-200 #{@class}"}>
+      <li :for={item <- @item} class="inline-flex items-center gap-x-2 py-2 text-sm font-medium text-slate-800">
```

Comment:
```suggestion
      <li :for={item <- @item} class="inline-flex items-center gap-x-2 py-2">
```
suggestion: Can we use default styling here and have consumers specify font styling via `class="text-sm font-medium text-slate-800"`? It should still work if applied to the parent element instead of each list item.

Reply:
alternately it looks like you _could_ do it [this way](https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#module-slot-attributes) to allow consumers to add classes to each item

```
slot :item do
  attr :class, :string, default: ""
end
```